packages/blob/src/index.ts

```diff
@@ -215,8 +211,58 @@ export async function list(
   };
 }
 
-function getApiUrl(): string {
-  return process.env.VERCEL_BLOB_API_URL || 'https://blob.vercel-storage.com';
+export interface GenerateClientTokenOptions extends BlobCommandOptions {
+  pathname: string;
+  onUploadCompleted?: {
+    callbackUrl: string;
+    metadata?: Record<string, string>;
```

Comment:
@correttojs metadata must be a string. We have no idea how folks will use it. Some will put objects in it, some simple strings, some query strings, ... Let's accept any string and delegate JSON.parse/JSON.stringify to the caller/receiver instead.

Reply:
why not `any` then?

Reply:
any would mean function/regex/dates are valid types wise, while when receiving the metadata, the format will no more work (a date would no more be a date unless we reparse the metadata..).

`string` is a low-level option that is flexible for all users without forcing them to send key/value when they don't need to.

Reply:
I'd still prefer a generic `TMetadata = any`, but I don't have a strong preference so `string` is also fine